build-docs-rsync

```diff
@@ -6,17 +6,40 @@ GIT_RESET="git reset --quiet --hard origin/master"
 cd /local/repos/phpdoc/phd
 $GIT_FETCH && $GIT_RESET >/tmp/log-phd 2>&1
 
-# Update docs sources
-cd /local/repos/phpdoc
-svn up >/tmp/log-svn 2>&1
+BASE_DIR=/local/repos/phpdoc-git
+cd $BASE_DIR
 
-# Build docs
 langs=`php -r 'include "/local/mirrors/phpweb/include/languages.inc"; echo implode( " ", array_keys( $ACTIVE_ONLINE_LANGUAGES ) );'`;
 
+if [ ! -d $BASE_DIR ]; then
+    mkdir $BASE_DIR;
+fi
+
+# Update the core module it self
+cd /home/docbuilder/phpdoc-git
+if [ ! -d $BASE_DIR/doc-base ]; then
+    git clone https://git.php.net/repository/doc/base.git $BASE_DIR/doc-base
+fi
+git -C $BASE_DIR/doc-base fetch origin && git -C $BASE_DIR/doc-base reset --hard origin/master
```

Comment:
The "clone if not exists then fetch and reset" task is repeated a bunch, perhaps make it a function?